CMSgov/mdct-mcr
# Working branch

## Description

Update AddEditModal to save a boolean value for CHIP checkbox
https://qmacbis.atlassian.net/browse/MDCT-1991?atlOrigin=eyJpIjoiNDk1N2Y5MmNkNzVkNGY1ZDgzNTcxODUzMzlhMzgxMWEiLCJwIjoiaiJ9

### How to test
Create a program, check the CHIP information checkbox and save. Check that 'combinedData' is stored as 'true'. Enter the program modal again and see that the checkbox has hydrated correctly. Uncheck the checkbox and see that the 'combinedData' field is stored as 'false'.

### Changed Dependencies


## Code author checklist
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests -- modified test setup to reflect combinedData turning into a boolean.

## Reviewer checklist (two different people)
